<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>gilgil</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>10w</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb50">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>50w</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb100">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>100w</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb500">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>500w</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>89</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refund">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>refund</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="w10l">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="w10">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>10w n</string>
   </property>
  </widget>
  <widget class="QLabel" name="w50">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>50w n</string>
   </property>
  </widget>
  <widget class="QLabel" name="w100">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>100w n</string>
   </property>
  </widget>
  <widget class="QLabel" name="w500">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>500w n</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="w50l">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="w100l">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="w500l">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
